ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/section_subject_domain_of_users .scs

```diff
@@ -0,0 +1,35 @@
+section_subject_domain_of_users 
```

Comment:
I think we can use already exist SD subject_domain_developers_actions_knowledge_base_cooperation_hierarchy. We will discuss about using current state of this SD for our propose. And let's discuss @ShunkevichDV why know we use users roles as norole relation?